knpu/episode3/protected-visibility.md

```diff
@@ -52,7 +60,9 @@ Back to the browser and refresh, and things still work!
 
 Let's temporarily add a new `private` function to `Ship` called `getSecretDoorCodeToTheDeathstar()`.
 Since only Empire ships should have access to this you can see why setting it as `private` makes sense.
-And let's return the secret code 'Ra1nb0ws'. 
+And let's return the secret code 'Ra1nb0ws':
+
+[[[ code('d3f3bf13b0') ]]]
 
 Over in `RebelShip` I should not be able to access this new function since we set it to `private`. 
```

Comment:
We're missing a code block right here showing us using this new function from inside `RebelShip` (I know, not obvious until you can see the video!)